tests/functional/events/11-cycle-task-event-job-logs-retrieve.t

```diff
@@ -17,7 +17,7 @@
 #-------------------------------------------------------------------------------
 # Test remote job logs retrieval, requires compatible version of cylc on remote
 # job host.
-export REQUIRE_PLATFORM='loc:remote'
```

Comment:
This would fail if run on a platform with a shared filesystem e.g. `_remote_background_shared_ssh`.